appveyor.yml

```diff
@@ -60,6 +60,9 @@ install:
     # Remove cygwin (and therefore the git that comes with it).
     - cmd: rmdir C:\cygwin /s /q
 
+    # Install workaround 64-bit support for VS 2008.
+    - cmd: if "%TARGET_ARCH%" == "x64" if "%CONDA_PY%" == "27" call .CI\vs2008\setup_x64.bat
```

Comment:
Everything else here is as it was when proposed to Eigen. Simply moved.

This is the only thing that was actually added. Simply to run this script on the right VM as part of the AppVeyor setup. This may need updating if/whenever [alternative CPython's]( https://github.com/conda-forge/conda-forge.github.io/issues/64 ) come to fruition.